Developers/Executing_Tests.rst

```diff
@@ -9,26 +9,27 @@ Executing our Tests
 coala has a big test suite. It is meant to work on every platform on
 every PC. If you just execute our tests you are doing us a favor.
 
-To run tests, You first need to install some dependencies.
-This can be done by following these steps:
+To run tests, you will need to open a file located in the default coala
+directory.
```

Comment:
too vague, which file, coala is not cloned yet (in principle). Also I don't recall someone needs to open some file :O